report-flat/src/App.vue

```diff
@@ -0,0 +1,70 @@
+<template>
```

Comment:
first of all, treat this PR as POC, besides that this flat-report is not intended to replace existing one, it have a different purpose (see description)

ad 1) it is open source project ;)
ad 2) IMO yes, it easier and cleaner for this particular purpose, (and you know it is the trendiest js framework... for today;) )
ad 3) another repo will only make things complex (don't forget about CI etc) besides that we already have 'modules' in various frameworks and even languages